shop-be/import-service/src/functions/importFileParser/handler.ts

```diff
@@ -0,0 +1,33 @@
+import { middyfy } from '@libs/lambda';
+import { GetObjectCommandInput, S3 } from '@aws-sdk/client-s3';
+import { S3Handler } from 'aws-lambda';
+import { REGION } from '@libs/constants';
+const csv = require('csv-parser')
+
+const importProductsFile: S3Handler = async (event) => {
+  const { Records } = event;
+  const s3 = new S3({ region: REGION });
+  let status = 200;
+  const res = []
+
+  try {
+    await Promise.all(Records.map(async element => {
+      const { s3: { object: { key } } } = element;
+      const params: GetObjectCommandInput = { Bucket: process.env.BUCKET_NAME, Key: key }
+
+      const object = await s3.getObject(params);
+
+      object.Body
+        .pipe(csv())
```

Comment:
Better to use pipeline with promise

Reply:
screwed up with it

Reply:
Didn't work or syntax was bad?

Reply:
didn't work, but I've solved problems in next task